### PR TITLE
Add ActiveRecord-backed fixtures.

### PR DESCRIPTION
**Pull request by beauby**
They are namespaced inside `ARModels` to avoid conficts with PORO models (that we might want to namespace as well).

### CODE REVIEW

File: test/fixtures/active_record.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    has_many :posts
+  end
+
+  class PostSerializer < ActiveModel::Serializer
+    attributes :id, :title, :body
+    params :title, :body
+
+    has_many :comments
+    belongs_to :author
+    url :comments
```
Comment (bf4) [inline test/fixtures/active_record.rb]: Didn't someone suggest a convention of something like

```ruby
class Post < AR::Base
  # stuff
  class PostSerializer < AM::Serializer
    # stuff
  end
end
```

**Comment by bf4** [comment]
:+1: except master doesn't have activerecord nor sqlite, right?

**Comment by beauby** [comment]
You're obviously right!

**Comment by bf4** [comment]
It sort of does by accident in dev... which is why I made https://github.com/rails-api/active_model_serializers/pull/1090

**Comment by beauby** [comment]
Fixed the deps.

File: Gemfile [hunk 2]
```diff
@@ -25,5 +25,11 @@ else
   gem 'actionpack', gem_version
 end
 
+group :test do
+  gem 'activerecord'
+  gem 'sqlite3', platform: :ruby
+  gem 'activerecord-jdbcsqlite3-adapter', platform: :jruby
+end
```
Comment (bf4) [inline Gemfile]: you're probably right that all the rails stuff above should be in a `gem :test` group..

File: test/fixtures/active_record.rb [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+    belongs_to :author
+  end
+
+  class AuthorSerializer < ActiveModel::Serializer
+    attributes :id, :name
+
+    has_many :posts
+  end
+end
```
Comment (bf4) [inline test/fixtures/active_record.rb]: would you be interested in something like https://github.com/rails-api/active_model_serializers/blob/master/test/poro_test.rb that just checks they're legit?

Comment (beauby) [inline test/fixtures/active_record.rb]: Yup, makes sense, I'll add it straight away.

Comment (bf4) [inline test/fixtures/active_record.rb]: ideally, I'd also want to test 'em all like i commented in https://github.com/rails-api/active_model_serializers/issues/878#issuecomment-96073030  (this is a pr i mean to make)

**Comment by joaomdmoura** [comment]
Cool @beauby this will be really helpful into new feature cc/ @NullVoxPopuli

**Pull request merged**